src/Raven.Server/ServerWide/Commands/ModifyDatabaseWatchersCommand.cs

```diff
@@ -64,14 +65,22 @@ public override string UpdateDatabaseRecord(DatabaseRecord record, long etag)
             if (Watcher == null)
                 return null;
 
-            var newTaskKey = Watcher.GetTaskKey().ToString();
-            if (Watcher.CurrentTaskId != null && Watcher.CurrentTaskId != newTaskKey)
+            var newTaskId = (Watcher.Database + Watcher.Url).ToLower();
```

Comment:
Don't do it like this. We don't need a task id in this manner.
Instead, just compare the two items when you need to do so. 
Otherwise the ids are going to look funny.